### ISSUE TITLE
[Module Proposal]: `avm/res/compute/disk-encryption-set`

### ISSUE THREAD

**Issue by krbar** [Azure/Azure-Verified-Modules#454]
### Check for previous/existing GitHub issues/module proposals

- [X] I have checked for previous/existing GitHub issues/module proposals.

### Check this module doesn't already exist in the module indexes

- [X] I have checked for that this module doesn't already exist in the module indexes; or I'm proposing the module to be migrated from CARML/TFVM.

### Bicep or Terraform?

Bicep

### Module Classification?

Resource Module

### Module Name

avm/res/compute/disk-encryption-set

### Module Details

Disk Encryption Set module, migration from CARML

### Do you want to be the owner of this module?

No

**Comment by segraef** [comment]
happy to own it

**Comment by matebarabas** [comment]
@krbar, @segraef, please agree internally on which path to choose:

Option 1: Kris will migrate this module, and once complete, Sebastian will become the owner.
Option 2: Sebastian becomes the owner right away and he'll migrate the module.

Please let me know the outcome in a comment here, and I'll help triage the issue.

Thank you both!

**Comment by segraef** [comment]
Just agreed with @krbar, option 1, thanks.

**Comment by krbar** [comment]
@matebarabas @segraef The PR (https://github.com/Azure/bicep-registry-modules/pull/749) is in review.
two remarks: 
- it looks like Sebastian is not yet marked as owner, so I had to add the ORPHANED.md to the module, otherwise it doesn't pass the static validation
- @segraef I've invited you as maintainer to both GH teams related to this module. As a creator, I am also maintainer in these teams. Feel free to remove me from the teams or change my team role once the module ownership is transferred to you
